### DISCUSSION TITLE
A university course that uses Mesa & Mesa-Geo

### DISCUSSION THREAD

**Discussion by wang-boyu** [mesa/mesa#1577] [General]
I was checking traffic to our repositories and noticed some visits from a referring site: [didawiki.cli.di.unipi.it](http://didawiki.cli.di.unipi.it). Apparently there is a course on Geospatial Analytics from The University of Pisa that introduced Mesa and Mesa-Geo: http://didawiki.cli.di.unipi.it/doku.php/geospatialanalytics/gsa/start

They even have a project assignment based on our GeoSchelling example in Mesa-Geo: http://didawiki.cli.di.unipi.it/lib/exe/fetch.php/geospatialanalytics/gsa/projects.pdf (see Project 19: Data-driven Geo Schelling model)

This is super cool! Would be interesting to know if they have any feedbacks on using Mesa and Mesa-Geo.

**Comment by tpike3** [comment]
That is awesome! I will reach out to them and add them to the list

**Reply by tpike3** [reply]
I just wrote them an introductory email

**Comment by pmchrist** [comment]
Hi! At University of Amsterdam we also use mesa in Agent-Based modeling class!

**Reply by wang-boyu** [reply]
Thanks for sharing!! @tpike3 Not sure which course this is but could be another university to add into your list?

**Reply by pmchrist** [reply]
Here is the page of a course, if it helps:
https://studiegids.uva.nl/xmlpages/page/2022-2023-en/search-course/course/99460

**Reply by wang-boyu** [reply]
This helps a lot! Thank you!

**Reply by tpike3** [reply]
@pmchrist Thanks!! I will reach out.

**Reply by tpike3** [reply]
I have reached out to Dr. Roy.

**Comment by JoeHelbing** [comment]
We're using mesa in an ABM class at UChicago as well in Computational Social Science. Just started a half personal interest/half class related dig into why mesa epstein_civil_violence seems to give different outputs than netlogo version with same parameters.

**Reply by JoeHelbing** [reply]
Update, was able to track down the issues and my fork of it produces the netlogo cyclical behaviors. The little victories.

**Reply by rht** [reply]
A PR to fix the example is appreciated!

**Reply by JoeHelbing** [reply]
Have never done one of those to a public git. Not sure where to start on that. Would be happy to contribute it if you happen to have a guide on how make sure it meets standards.

**Reply by JoeHelbing** [reply]
I think I found it. https://github.com/projectmesa/mesa/blob/912dfbf73088fbe0796a68e629f425e100a202bc/CONTRIBUTING.rst

Will go through it and make sure it meets these.

**Reply by tpike3** [reply]
Awesome, thanks!

**Reply by tpike3** [reply]
I have reached out to Dr. Clipperton
